modules/Data.py

```diff
@@ -106,3 +106,12 @@ def get_lending_currencies():
     for cur in lending_balances:
         currencies.append(cur)
     return currencies
+
+
+def truncate(f, n):
```

Comment:
It is from the ever helpful stackexchange https://stackoverflow.com/questions/783897/truncating-floats-in-python

Would you suggest I instead multiply the number by 100,000,000, floor(), then divide by 100,000,000.0?